pegasus/sites.v3/hourofcode.com/public/beyond.haml

```diff
@@ -1,39 +1,58 @@
 ---
 title: Beyond an Hour of Code
-layout: wide
+layout: wide_index
 ---
 
-%link{href: "/css/beyond.css", rel: "stylesheet"}
-
-%h1 Beyond an Hour of Code
-
-%h1#teacher-section For teachers
-%p
-  You've held an Hour of Code event in your classroom... now what? Join the more than 2 million educators who use Code.org to teach computer science. Anyone can learn!
-.half-resource-block{style: "margin-right: 20px;"}
-  %img.top-image{src: "/images/beyond-teacher.jpg"}
-  .description-area
-    .description
-      %h2.resource-title Teach With Code.org’s Courses
-      Code.org offers a full K-12 course pathway and professional learning workshops for teachers, free of charge.
-    %a{href: "https://code.org/teach"}
-      %button.gray-button Teach with Code.org
-
-.half-resource-block
-  %img.top-image{src: "/images/beyond-courses.png"}
-  .description-area
-    .description
-      %h2.resource-title Teach Courses from Other Providers
-      Compare options by different partners, organized by grade band.
-    %a{href: "https://code.org/educate/curriculum/3rd-party"}
-      %button.gray-button Explore other providers
-
-.clear
-
-%h1#student-section For students
-%p
-  Want to keep learning after Hour of Code? Check out these activities for learners of all ages and skill levels.
-
-%div
-  = view :learn_carousels, variation:'beyond'
+%link{href: "/css/generated/hoc-banner.css", rel: "stylesheet", type: "text/css"}
+%link{href: '/css/full-width-section.css', rel:'stylesheet', type:'text/css'}
+%link{href: "/css/generated/beyond-carousel.css", rel: "stylesheet", type: "text/css"}
+
+#fullwidth
+  = view :header
+
+  %section.banner
+    .wrapper
+      %h1
+        =hoc_s(:beyond_an_hour)
+      %p.body-one
+        =hoc_s(:hoc_beyond_hero_description)
+
+%section
+  .wrapper
+    .section-intro.centered
+      %h2.centered 
```

Comment:
Can you add the same `.heading-xxl` class here that's mentioned in hoc_beyond_for_teachers_section_title?